action.yml

```diff
@@ -281,13 +329,17 @@ runs:
          fi
 
          cd kernel/${{ inputs.kernel-dir }}
+         if ! grep -qsi "LD *= *ld.lld" Makefile; then
+             if [ -f "$HOME/clang/bin/ld" ]; then
+                 rm -f "$HOME/clang/bin/ld"
+             fi
+         fi   
          function version_gt() { test "$(echo -e "$1\n$2" | sort -V | tail -n1)" == "$1"; }
          VERSION=$(grep -E '^VERSION = ' Makefile | awk '{print $3}')
          PATCHLEVEL=$(grep -E '^PATCHLEVEL = ' Makefile | awk '{print $3}')
-         SUBLEVEL=$(grep -E '^SUBLEVEL = ' Makefile | awk '{print $3}')
```

Comment:
为什么删掉这行？

Reply:
> 为什么删掉这行？

SUBLEVEL这个变量是多余的，对于我的内核4.9.25判断前两个数字4和9就可以了，后面的数字25不需要判断

Reply:
```bash
 Initializing KernelSU
  Kernel version: 4.19.289
  Warning: The KernelSU version you selected was detected to be main, but KernelSU has dropped support for the non-gki kernel since 0.9.5. 
   This will force switch to v0.9.5.
  [+] Setting up KernelSU...
  Cloning into 'KernelSU'...
```